pages/demo/html/elementul-br.tsx

```diff
@@ -5,34 +5,24 @@ import Demo, { DemoPreview } from '~/components/demo';
 import Header from '~/components/Header';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.